tests/test_usgs.py

```diff
@@ -0,0 +1,161 @@
+                                {"value": '112', "dateTime": "2020-07-02T00:00:00.000"}
+                            ]}],
+                        "sourceInfo": {
+                            "geoLocation": {
+                                "geogLocation":
+                                    {"longitude": -106.54, "latitude": 37.35}
+                            },
+                            "timeZoneInfo": {"defaultTimeZone": {"zoneOffset": "-5:00"}}
+                        },
+                        "variable": {"unit": {"unitCode": "cf/s"}}
+                    }]}
+        }
+
+    @classmethod
+    def crp_side_effect(cls, url, **kwargs):
+        mock = MagicMock()
+        params = kwargs["params"]
+
+        if "startDT" not in params:
+            mock.json.return_value = cls.crp_meta_return()
+        elif "startDT" in params:
+            mock.json.return_value = cls.usgs_daily_discharge_response()
+        else:
+            raise ValueError("unknown scenario")
+
+        return mock
+
+    def test_class_variables(self):
+        assert USGSPointData("no", "no").tzinfo == timezone(timedelta(hours=0.0))
+
+    def test_get_metadata(self, crp_station):
+
+        with patch("metloom.pointdata.usgs.requests") as mock_requests:
+            mock_requests.get.side_effect = self.crp_side_effect
+            metadata = crp_station.metadata
+            mock_get = mock_requests.get
+            assert mock_get.call_count == 1
+            mock_get.assert_called_with(
+                "https://waterservices.usgs.gov/nwis/dv/",
+                params={
+                    'sites': '08245000',
+                    'format': 'json',
+                    'siteType': 'ST',
+                    'siteStatus': 'all'
+                },
+            )
+
+        expected = gpd.points_from_xy([-106.54], [37.35])[0]
+        assert expected == metadata
+
+    def test_get_daily_data(self, crp_station, crp_daily_expected):
+        with patch("metloom.pointdata.usgs.requests") as mock_requests:
+            mock_get = mock_requests.get
+            mock_get.side_effect = self.crp_side_effect
+            response = crp_station.get_daily_data(
+                datetime(2020, 7, 1),
+                datetime(2020, 7, 2),
+                [USGSVariables.DISCHARGE],
+            )
+
+            mock_get.assert_any_call(
+                "https://waterservices.usgs.gov/nwis/dv/",
+                params={
+                    'startDT': datetime(2020, 7, 1).date().isoformat(),
+                    'endDT': datetime(2020, 7, 2).date().isoformat(),
+                    'sites': '08245000',
+                    'parameterCd': '00060',
+                    'format': 'json',
+                    'siteType': 'ST',
+                    'siteStatus': 'all'
+                },
+            )
+            assert mock_get.call_count == 2
+        pd.testing.assert_frame_equal(response, crp_daily_expected)
```

Comment:
These look great!

Could we add tests for
- daily data call with no data available
- points from geometry call